powertools-parameters/pom.xml

```diff
@@ -77,6 +77,11 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>dynamodb</artifactId>
         </dependency>
+        <dependency>
```

Comment:
General comment, not to solve here: We're adding a new dependency, making the parameters module heavier (we did it already with DynamoDB). 

Should we think about different parameter modules (one per persistence technology: param store, DDB, app config, secrets manager) to avoid a big one with everything inside?

Note: parameter module is 29KB in v1.14, 34KB in v1.15 (with the add of DDB), how much with AppConfig?
29 or 39kb is not a big deal but just to discuss...

Reply:
@msailes your thoughts on this?

Reply:
Given the variety of sources that could be introduced, I think this makes sense for a v2 ticket.

Reply:
My feeling is we should nest modules again. The parameters modules should all standalone (apart from the base interface pieces of course); may as well let people pull them in separately. ACK that it feels a bit much for this change (and would be interface breaking for existing params providers)